src/main.rs

```diff
@@ -525,9 +534,10 @@ fn run() -> Result<(), failure::Error> {
 
         match kv_matches.subcommand() {
             ("create", Some(create_matches)) => {
-                let target = manifest.get_target(create_matches.value_of("env"), false)?;
-                let title = create_matches.value_of("title").unwrap();
-                commands::kv::namespace::create(&target, user, title)?;
+                let env = create_matches.value_of("env");
+                let target = manifest.get_target(env, false)?;
+                let binding = create_matches.value_of("binding").unwrap();
```

Comment:
`unwrap` here - can we do a match on `binding` and print a nice error message if it's excluded? or put in `unreachable` if `clap` prevents this from ever being `None`?

Reply:
We're alright to use unwrap because this value is already guaranteed to exist by Clap :)